packages/synapse-react-client/src/components/EntityCitation/EntityCitation.test.tsx

```diff
@@ -0,0 +1,290 @@
+  setSuccess: setMockUseGetEntityBundleSuccess,
+} = getUseQueryMock<Partial<EntityBundle>, SynapseClientError>()
+
+const { mock: useGetEntityDOIMockImpl, setSuccess: setEntityDOISuccess } =
+  getUseQueryMock<DoiAssociation | null, SynapseClientError>()
+
+const { mock: useGetProjectDOIMockImpl, setSuccess: setProjectDOISuccess } =
+  getUseQueryMock<DoiAssociation | null, SynapseClientError>()
+
+const fileWithAnnotations: Partial<EntityBundle> = {
+  entity: {
+    id: 'syn61841662',
+    name: 'blackcat.jpg',
+    concreteType: 'org.sagebionetworks.repo.model.FileEntity',
+  },
+  doiAssociation: {
+    objectId: 'syn61841662',
+    objectType: DoiObjectType.ENTITY as ObjectType,
+    doiUri: '10.7303/syn61841662',
+    associationId: '',
+    etag: '',
+    doiUrl: '',
+    associatedBy: '',
+    associatedOn: '',
+    updatedBy: '',
+    updatedOn: '',
+  },
+  entityType: EntityType.FILE,
+}
+
+const doiEntitySuccess: DoiAssociation = {
+  objectId: 'syn61841662',
+  objectType: DoiObjectType.ENTITY as ObjectType,
+  doiUri: '10.7303/syn61841662',
+  associationId: '',
+  etag: '',
+  doiUrl: '',
+  associatedBy: '',
+  associatedOn: '',
+  updatedBy: '',
+  updatedOn: '',
+}
+
+const versionedDoiEntitySuccess: DoiAssociation = {
+  objectId: 'syn66268092',
+  objectType: DoiObjectType.ENTITY as ObjectType,
+  doiUri: '10.7303/syn66268092.1',
+  associationId: '',
+  etag: '',
+  doiUrl: '',
+  associatedBy: '',
+  associatedOn: '',
+  updatedBy: '',
+  updatedOn: '',
+}
+
+const doiProjectSuccess: DoiAssociation = {
+  objectId: 'syn64042437',
+  objectType: DoiObjectType.ENTITY as ObjectType,
+  doiUri: '10.7303/syn64042437',
+  associationId: '',
+  etag: '',
+  doiUrl: '',
+  associatedBy: '',
+  associatedOn: '',
+  updatedBy: '',
+  updatedOn: '',
+}
+
+const openPopover = async (buttonName: string) => {
+  const button = screen.getByRole('button', { name: `${buttonName}` })
+  await userEvent.click(button)
+}
+
+describe('EntityCitation tests', () => {
+  beforeEach(() => {
+    vi.clearAllMocks()
+    mockUseGetEntityBundle.mockImplementation(useGetEntityBundleMockImpl)
+  })
+
+  it('renders "Cite page" button when only entity DOI exists', async () => {
+    mockUseGetDOIAssociation.mockImplementation((request, options) => {
+      if (request.id === 'syn61841662' && request.version === 1) {
+        return useGetEntityDOIMockImpl()
+      } else if (
+        request.id === 'syn60582629' &&
+        request.version === undefined
+      ) {
+        return useGetProjectDOIMockImpl()
+      }
+      return useGetEntityDOIMockImpl()
+    })
+
+    render(
+      <EntityCitation
+        projectId="syn60582629"
+        entityId="syn61841662"
+        versionNumber={1}
+      />,
+      { wrapper: createWrapper() },
+    )
+
+    act(() => {
+      setMockUseGetEntityBundleSuccess(fileWithAnnotations)
+      setEntityDOISuccess(doiEntitySuccess)
+      setProjectDOISuccess(null)
+    })
+
+    openPopover('Cite page')
```

Comment:
You should be getting a lint error here?
```suggestion
    await openPopover('Cite page')
```